web/components/ResourceBar/ResourceBar.vue

```diff
@@ -32,6 +32,8 @@
     <template v-if="!editmode">
       <SchedulingResults />
       <ResourceDefinitionTree :items="treeData" />
+      <!-- This is need to avoid bug on vuetify -->
```

Comment:
This is required to work around the vuetify's bug, refer more details in https://github.com/kubernetes-sigs/kube-scheduler-simulator/issues/10

Reply:
@haosdent 
Thanks for the suggestion. 
Fixed. 🙏